uwsgi-example/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# We have to call 'hooks/run' so that we pickup its workarounds instead of
+# calling 'uwsgi' directly, or we could duplicate the workarounds here.
+exec "$(hab pkg path tvaughan/uwsgi)/hooks/run"                               \
```

Comment:
What's the right way to call a package from another package? I naively tried `hab start tvaughan/uwsgi` but this doesn't accept command-line arguments.